source/simulator/parameters.cc

```diff
@@ -126,6 +126,14 @@ namespace aspect
                        "here, one can choose the time step as large as one wants (in particular, "
                        "one can choose $c>1$) though a CFL number significantly larger than "
                        "one will yield rather diffusive solutions. Units: None.");
+
+    prm.declare_entry ("Use smoothing", "false",
+                       Patterns::Bool (),
+                       "If set to false, the artificial viscosity of a cell is computed and used "
+                       "like normal.  If set to true, the maximum of the artificial viscosity in "
```

Comment:
"like normal" is not very informative. Maybe "is computed on every cell separately as discussed in \\cite{KHB12}".